### ISSUE TITLE
We encountered a problem with your local email database. Error: spawn /usr/share/mailspring/resources/app.asar.unpacked/mailsync ENOENT  We will now attempt to rebuild it.

### ISSUE THREAD

**Issue by erikkallen** [Foundry376/Mailspring#18]
When I try to start the application all I get is an error message box with the following error:

We encountered a problem with your local email database. Error: spawn /usr/share/mailspring/resources/app.asar.unpacked/mailsync ENOENT

We will now attempt to rebuild it.

##### Are there any related issues?

No
##### What operating system are you using?
Ubuntu 17.04

##### What version of Mailspring are you using?
1.0.0 linux 64
--

**Bug?**
##### Do you have any third-party plugins installed? If so, which ones?
no none

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
No this is on startup

##### Is the issue reproducible with a particular attachment, message, signature, etc?

No

**Feature Request?**
##### Does this feature exist in another mail client or tool you use?
No

**Comment by letic** [comment]
Same here on Debian Sid.

Tried to strace/gdb it but didn't get any interesting info.

It's worth noting that the message keeps coming and that processes needs to be killed with a 9 signal as a 15 is not enough.

Let me know if I can help debugging.

Cheers
LeTic

**Comment by ldartez** [comment]
Hello, I think this is a duplicate of issue #14.

**Comment by bengotow** [comment]
Hey folks—thanks for checking out Mailspring! The Linux version actually hasn't been released yet 
 (sorry if there's a download link live somewhere!) This issue and a few others need to get worked out before we can ship it. Gonna close this as a dup of #14.
